src/main/java/redis/clients/jedis/JedisClientConfig.java

```diff
@@ -80,4 +80,18 @@ default HostAndPortMapper getHostAndPortMapper() {
     return null;
   }
 
+  public interface ClientSetInfoConfig {
```

Comment:
Should we use a separate inner interface for two fields? Or, do we add those two fields directly into existing interface?

@yangbodong22011 @uglide

Reply:
I agree with putting it together, but I want to add a separate class called `ClientSetInfoConfig` instead of putting it in `JedisClientConfig`. In detail, I want the builder interface to be:
```
try (Jedis jedis = new Jedis(hnp, DefaultJedisClientConfig.builder().password("foobared")
    .clientSetInfoConfig(ClientSetInfoConfig.builder().disabled().build()).build())) {
```